### ISSUE TITLE
[3.8.366] unbumped apt package dependencies

### ISSUE THREAD

**Issue by Matthew-Beckett** [Matthew-Beckett/netxms-dockerfiles#6]
Error thrown

```bash
The following packages have unmet dependencies:
 netxms-agent : Depends: netxms-base (= 3.8.314-1) but 3.8.366-1 is to be installed
 netxms-dbdrv-sqlite3 : Depends: netxms-base (= 3.8.314-1) but 3.8.366-1 is to be installed
 netxms-server : Depends: netxms-base (= 3.8.314-1) but 3.8.366-1 is to be installed
```

when running

MAJOR_VERSION=3.8
MINOR_VERSION=366

```bash
apt-get update && apt-get -y install netxms-server="${MAJOR_VERSION}.${MINOR_VERSION}-1" netxms-agent="${MAJOR_VERSION}.${MINOR_VERSION}-1" netxms-dbdrv-sqlite3="${MAJOR_VERSION}.${MINOR_VERSION}-1
```

**Comment by Matthew-Beckett** [comment]
@2128506 are you able to take a look at this? I think the dependencies in the 3.8.366 package have not been changed to target 3.8.366 from 314.

**Comment by 2128506** [comment]
Hi!
I have asked Alex to take a look, he knows how deb packages are built.
Do you have Telegram account? We could setup Telegram group for easier communications then.
Best regards,
Victor

**Comment by Matthew-Beckett** [comment]
Hi Victor,

I do, I will PM your my number on the NetXMS Forum.

Thanks for involving Alex, I would quite like to get the deb package dependencies fixed as opposed to installing `netxms-base=3.8.314` on the 3.8.366 image.
